source/component/PasDoc_GenHtml.pas

```diff
@@ -2333,6 +2335,20 @@ function TGenericHTMLDocGenerator.FormatItalic(const Text: string): string;
   Result := '<em>' + Text + '</em>';
 end;
 
+function TGenericHTMLDocGenerator.FormatWarning(const Text: string): string;
+begin
+  Result := '<dl class="tag warning"><dt>' + FormatBold('Warning') + '</dt><dd>';
```

Comment:
The strings 'Warning' and 'Note' should go through our translation system.

( I know, it's a big pain to add a new translation keyword, because you have to edit all translation files to compile... I really wish we should change it, https://github.com/pasdoc/pasdoc/issues/87 . But for the time being, we have to work with what we have :) )